js/CameraRoll.js

```diff
@@ -100,7 +100,10 @@ export type PhotoIdentifiersPage = {
     end_cursor?: string,
   },
 };
-
+export type SaveToCameraRollOptions = {
+  type?: 'photo' | 'video' | 'auto',
```

Comment:
this new 'auto' type is useful, but shouldn't be in the same PR. It's much easier to assess impact and approve PR's when there is exactly one feature proposed and implemented.

Reply:
To be honest i just added auto to use default values instead of checking for undefined, so i didn't thought that as a feature 😆